File: fbc-portal_cs/src/app/shared/interfaces/artigo.ts
import { Unidade } from './unidade';

export interface Artigo {
    Codigo: string;
    Descricao?: string;
    CodBarras?: string;
    Marca?: string;
    Modelo?: string;
    UnidadeVenda?: string;
    PVP1?: number;
}

File: fbc-portal_ss/fbc_webapi/Models/Artigo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace fbc_webapi.Models
{
    public class Artigo
    {
        public string Codigo { get; set; }
        public string Marca { get; set; }
        public string Modelo { get; set; }
        public string CodBarras { get; set; }
        public string Descricao { get; set; }
        public string UnidadeVenda { get; set; }
        public double? PVP1 { get; set; }
    }
}